slime/rollout/sglang_example.py

```diff
@@ -81,20 +89,48 @@ async def generate(args, sample: Sample, sampling_params) -> Sample:
         sample.status = Sample.Status.TRUNCATED
         return sample
 
-    # Handle partial rollout samples: continue generation from existing response
-    input_text = sample.prompt + sample.response
+    if state.use_token_output:
```

Comment:
we have `args` passed to this function, so `args.use_token_output` would be better.